packages/plugin-user/src/user.js

```diff
@@ -173,6 +189,55 @@ const User = SparkPlugin.extend({
   },
 
   /**
+   * Sets password status on token
+   * @param {Boolean} value
+   * @returns {undefined}
+   */
+  setPasswordStatus(value) {
+    if (this.spark.credentials && this.spark.credentials.supertoken) {
```

Comment:
this should call a credentials method rather than talking to supertoken directly. that'll make it easier to gut the credentials plugin(s) later.